src/test/java/com/datasonnet/XMLWriterTest.java

```diff
@@ -200,6 +168,28 @@ void testOmitXml() throws Exception {
         assertTrue(mappedXml.startsWith("<?xml"));
     }
 
+    @Test
+    void testXMLRoot() throws Exception {
+        String jsonData = TestResourceReader.readFileAsString("xmlRoot.json");
+
+        Mapper mapper = new Mapper("DS.Formats.write(payload, \"application/xml\")", new ArrayList<>(), true);
+        try {
+            String mappedXml = mapper.transform(new StringDocument(jsonData, "application/json"), new HashMap<>(), "application/xml").contents();
+            fail("Must fail to transform");
+        } catch(IllegalArgumentException e) {
+            assertTrue(e.getMessage().contains("Object must have only one root element"), "Found message: " + e.getMessage());
+        }
+
+        mapper = new Mapper("local params = {\n" +
+                "    \"RootElement\" : \"TestRoot\",\n" +
```

Comment:
Okay, I see, so this is an output convenience to avoid someone having to write `{"root": {"inside": "stuff"}}`? Sure, why not